README.md

```diff
@@ -138,7 +167,31 @@ spec:
       protocol: TCP
       port: 443
       targetPort: 443
-EOF
 
+```bash
 $ kubectl apply -f ingress-service.yaml
 ```
+
```

Comment:
As an afterthought @rbarry82 I was wondering if we do need to put into our README all these suggestions to use Metallb, Ingress, and this Kubernetes patch. I was a bit overwhelmed by this. Is there any way to simplify this ?

Reply:
I wonder whether the ingress is actually needed at all. I deployed it at first, since the initial docs I started from indicated it, but given that Prometheus is only "reaching out", it may be possible to just exclude this entirely.

Reply:
Removed after discussion.